src/module_1/module_1_meteo_api.py

```diff
@@ -9,29 +12,87 @@
 }
 VARIABLES = ["temperature_2m_mean", "precipitation_sum", "wind_speed_10m_max"]
 START_DATE = "2010-01-01"
-END_DATE = "2020-12-31"
+END_DATE = "2020-01-02"
```

Comment:
Very good job with the magic variables. Always capitalized and at the beginning of the file.